site/en/blog/prerender-pages/index.md

```diff
@@ -0,0 +1,263 @@
+
+### Measuring prerenders
+
+Whether a page is prerendered can be seen with a non-zero activationStart entry of [PerformanceNavigationTiming](https://developer.mozilla.org/docs/Web/API/PerformanceNavigationTiming), supplemented with changes for some navigation currently not reported by that. This can then be logged using a Custom Dimension, or similar when logging the page views:
+
+```js
+// Set Custom Dimension for Prerender status
+const prerendered =
+    performance &&
+    performance.getEntriesByType &&
+    performance.getEntriesByType('navigation')[0] &&
+    performance.getEntriesByType('navigation')[0].activationStart > 0;
+gtag('set', {'dimension1': prerendered});
+// Initialise GA - including sending page view by default
+gtag('config', 'UA-12345678-1');
+```
+
+This will allow your analytics to show how many navigation are prerendered compared to other types of navigation, and also allow you to correlation any performance metrics or business metrics to these different navigation types. Faster pages means happier users, which can often have real impact on business measures as our [case studies](https://web.dev/tags/case-study/) show.
+
+As you measure the business impact of prerendering pages for instant navigations, you can decide whether it is worth investing more effort in using this technology to allow more navigations to be prerendered, or to investigate why pages are not being prerendered.
+
+#### Measuring hit rates
+
+As well as measuring the impact of pages that are visited after a prerender, it is also important to measure pages that are prerendered and _not_ subsequently visited. This could imply you are prerendering too much, and using up valuable resources of the user for little benefit.
+
+This is measured by firing an analytics event when speculation rules are inserted. You can then compare the number of these events, to the actual prerender page views. Or alternatively fire another event (`prerender-activated`) on activation if that makes it easier to compare.
+
+The “successful hit rate” can then be calculated by looking at the difference between these two figures. For pages where you are using the Speculation API to prerender the pages, you can adjust the rules appropriately to ensure you keep a high hit rate to maintain the balance between using up the users resources to help them, versus using it needlessly.
+
+Be aware that some prerendering may be taking place due to the omnibox prerendering and not just your speculation rules. You can check the `document.referrer` (which will be blank for omnibox navigation including prerendered omnibox navigations) if you wish to differentiate these.
```

Comment:
Calling out the inconsistent capitalization of "omnibox" here.

Reply:
FWIW, it looks like the term is lowercase in the developer-facing docs I see the term used in.